index.html

```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
         <ul class="skills">
           <li>html</li>
+          <li>Ruby on rails</li>
           <li>css</li>
           <li>Javascript</li>
         </ul>
         <div class="but"><button class="button">See project</button></div>
       </div>
-      
+    </div>
+  </section>
 
-      <section class="about-myself">
-        <div class="introduction">
-          <h1>About<br>Myself</h1>
-          <p class="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
-          <a href="#" class="connect">LET'S CONNECT</a><br><br>
-           <div class="socials">
-           <ul>
+
+    <section class="about-myself">
+      <div class="introduction">
+        <h1 class="self">About Myself</h1>
+        <p class="about">Hello I'm a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don't
+          hestiate to contact me.</p><br>
+        <a href="#" class="connect1">LET'S CONNECT</a><br><br>
+
+        <div class="socials1">
+          <ul>
             <li><a href=""><img src="./images/github.png" alt="github"></a></li>
             <li><a href=""><img src="./images/linkedin.png" alt="linkedin"></a></li>
             <li><a href=""><img src="./images/v.png" alt="peace"></a></li>
             <li><a href=""><img src="./images/twitter.png" alt="twitter"></a></li>
             <li><a href=""><img src="./images/m.png" alt="mail"></a></li>
-           </ul><br>
-            <button class="resume">Get my resume</button> <br><br><br>
-           </div>
-           <div class="lang-skills">
-             <span class="lg">Languages</span>
-             <img class="down" src="./images/10.png" ></img>
+          </ul><br>
+          <button class="resume">Get my resume</button> <br><br><br>
+        </div>
+      </div>
+
+      <div>
+        <div class="lang-skills">
+          <span class="lg">Languages</span>
+          <img class="down" src="./images/10.png"></img>
         </div>
         <ul class="show-skills">
           <li class="sl">
-            <img class="lang-img" src="./images/js.png" alt="javascript"> 
+            <img class="lang-img" src="./images/js.png" alt="javascript">
             <span class="span">JavaScript</span>
           </li>
           <li class="sl">
-            <img class="lang-img" src="./images/html.png" alt="html"> 
+            <img class="lang-img" src="./images/html.png" alt="html">
             <span class="span">HTML</span>
           </li>
           <li class="sl">
-            <img class="lang-img" src="./images/css.png" alt="css"> 
+            <img class="lang-img" src="./images/css.png" alt="css">
             <span class="span">CSS</span>
           </li>
         </ul>
```

Comment:
- The mobile version of the `about-myself` section seems to have a bug, please fix this 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
<img width="234" alt="mxfmx" src="https://user-images.githubusercontent.com/73607512/221367574-58754608-1eca-46fe-aef1-1fc611010719.PNG">